chameleon-server/src/main/java/com/chameleonvision/common/hardware/GPIO.java

```diff
@@ -0,0 +1,48 @@
+package com.chameleonvision.common.hardware;
+
+import com.pi4j.io.gpio.GpioController;
+import com.pi4j.io.gpio.GpioFactory;
+import com.pi4j.io.gpio.GpioPinDigitalOutput;
+import com.pi4j.io.gpio.RaspiPin;
+
+public class GPIO {
```

Comment:
This can't be Pi-specific. As an interface implementation for Pi, this is fine. But any other SBC is now at a disadvantage.